frontends/web/src/routes/account/utils.ts

```diff
@@ -100,6 +100,12 @@ export type TAccountsByKeystore = {
   accounts: IAccount[];
 };
 
+export const sortAccounts = (accounts: IAccount[]): IAccount[] => {
+  return Array.from(accounts).sort((ac1, ac2) => {
+    return `${ac1.keystore.rootFingerprint}${ac1.code}`.localeCompare(`${ac2.keystore.rootFingerprint}${ac2.code}`);
```

Comment:
@benma should we rely on that Account.code will always include rootFingerprint in the future?

In that case this could be simplified by just: `ac1.code.localeCompare(ac2.code})`

Reply:
I'll keep as is, we should assume it will change.